.gitignore

```diff
@@ -11,3 +11,4 @@ build
 gradlew.bat
 *.properties
 src/main/res/
+MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/account/AccountRetrievalService.java
```

Comment:
Why is this being added to the gitignore file? Any chance you could add a comment above it explaining what the file is and why it's not being checked in to git?

Reply:
Removed from gitignore file. Was part of an experiment 🔬